### PR TITLE
fix: paragraph style missing in contribution page

### LINKED ISSUE #4601: Paragraph style missing in contribution page
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

In the contribution page everything is center except paragraph(**justified**)

Screenshot from 2024-01-15 11-45-58

### Steps To Reproduce

footer->contribute->see bug

### Expected Behavior

We have to make it center 

Screenshot from 2024-01-15 11-57-05

### Screenshots

_No response_

### Device Information [optional]

```markdown
- OS:
- Browser:
- version:
```

### Additional context

_No response_

### Are you working on this issue?

No

### PR DESCRIPTION
**Pull request by ManeetChhabra**
Fixes #4601 

#### Describe the changes you have made in this PR -
center aligned text on contribution page from justified.

### Screenshots of the changes (If any) -

image

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by ManeetChhabra** [comment]
@Malavi1 @tanmoysrt have a look...

**Comment by amananand20** [comment]
hey @ManeetChhabra, you can even use bootstrap class(text-center) to make text align center as bootstrap is being used in this project.

File: app/assets/stylesheets/contribute.scss [hunk 1]
```diff
@@ -65,6 +65,10 @@
   }
 }
 
+.justified-paragraph {
+  text-align: center;
+}
+
```
Comment (tanmoysrt) [inline app/assets/stylesheets/contribute.scss:71]: the class name should be like `centered-paragraph`, as it has property `text-align: center`

**Review by tanmoysrt — CHANGES_REQUESTED** [review]

**Comment by ManeetChhabra** [comment]
@tanmoysrt sir, i made the required changes.

**Review by tanmoysrt — APPROVED** [review]
Working as expected :+1: 
Good job @ManeetChhabra

**Comment by ManeetChhabra** [comment]
> Working as expected 👍 Good job @ManeetChhabra

Thanks @tanmoysrt !!

**Pull request merged**